introduction/Organization.tex

```diff
@@ -6,39 +6,100 @@
 %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Organization}
 
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Executive Committee
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Executive Committee --- ec@robocupathome.org}
 \label{sec:ec}
-The \iaterm{Executive Committee}{EC} consists of members of the board of trustees, and representatives of each activity area. Members representing the @Home league:
+The \iaterm{Executive Committee}{EC} consists of members of the board of trustees, and representatives of each activity area. The EC is responsible of the direction of the RoboCup@Home leagues, and has the last word in any decision taken. Is duty of the EC to instruct the new TC members when they join the committee, as well as instruct the OC Chair and supervise the decisions of both committees.
+
+\subsubsection{Members representing the @Home league}
 \begin{itemize}
   \item Kai Chen (University of Science and Technology of China, China)
   \item Dirk Holz (University of Bonn, Germany)
-  \item Caleb Rascon (Universidad Nacional Autonoma de Mexico, Mexico)
+  \item Caleb Rascon (Universidad Nacional Autónoma de México, Mexico)
   \item Sven Wachsmuth (Bielefeld University, Germany)
 \end{itemize}
 
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Technical Committee
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Technical Committee --- tc@robocupathome.org}
 \label{sec:tc}
-The \iaterm{Technical Committee}{TC} is responsible for the rules of each league. Members of the RoboCup@Home Technical Committee for \YEAR:
+The \iaterm{Technical Committee}{TC} is responsible for the rules of each league. Its responsibilities include:
+
+\begin{enumerate}
+  \item During preparation time
+  \begin{itemize}
+    \item Write and buff the rules in the rulebook
+    \item Design and write new tests (when applicable)
+    \item Evaluate previous performance and update tests accordingly
+  \end{itemize}
+  \item During the competition
+  \begin{itemize}
+    \item Direct Q\&A sessions during Team Leader meetings
+    \item Renounce to their teams and serves as referees (see \refsec{rule:referees})
+    \item Take decisions about partial scores and penalties
+  \end{itemize}
+\end{enumerate}
+
+The \iaterm{Technical Committee}{TC} is renewed every year, being its members elected by the participating teams. However, elected members still have to earn their place in the TC during the running year by actively contributing with the next-year's rulebook creation. The Executive Committee reserves to itself the legitimate right of replacing inactive TC members with more enthusiastic and participative people from the league who has been actively contributing during the year. It is advised that candidates work with the \iaterm{Organizing Committee}{OC} for one year before trying to join the Technical Committee.
```

Comment:
OC before TC is new to me. Perhaps Contributing to the rulebook is also a precursor to TC-ing?

Reply:
Advised but not mandatory. As of 2009~2012 was mentioned by Execs right before the elections.